lib/matplotlib/colors.py

```diff
@@ -1225,6 +1254,876 @@ def reversed(self, name=None):
         return new_cmap
 
 
+class MultivarColormap:
+    """
+    Class for holding multiple `~matplotlib.colors.Colormap` for use in a
+    `~matplotlib.cm.VectorMappable` object
+    """
+    def __init__(self, name, colormaps, combination_mode):
+        """
+        Parameters
+        ----------
+        name : str
+            The name of the colormap family.
```

Comment:
This is true, but I am not sure if it is common for users to add custom colormaps to the registry.
(also `ColormapRegistry.register()` takes `name` as a keyword argument, so you can technically use a different name in the registry).

However, there is something else we can discuss here.
I chose to base the call signature of `BivarColormap` on the call signature of `Colormap` for concistency:
 ```py
 class Colormap:
    def __init__(self, name, N=256):
 ```
Here `name` is the first argument. 
However, since a unique name is not required to use a custom colormap, it might be better to have this as a keyword argument. 
I also just noticed that there is an inconsistency between the existing colormap subclasses:
 ```py
 class LinearSegmentedColormap(Colormap):
    def __init__(self, name, segmentdata, N=256, gamma=1.0):
 class ListedColormap(Colormap):
    def __init__(self, colors, name='from_list', N=None):
 ```
 
 We **could** have a call signature for `BivarColormap` with `name` as a keyword argument:
 ```py
class MultivarColormap:
    def __init__(self, colormaps, combination_mode, name='multivariate_colormap'):
 ```
 instead of the existing
 ```py
class MultivarColormap:
    def __init__(self, name, colormaps, combination_mode):
 ```
 and I think having 'name' as a keyword argument would be slightly easier for the user.
 What do you think?